Undefined reference to inline function

In Debian we've seen [a build failure](https://bugs.debian.org/972372) on 32bit platforms with GCC-10:

```
Extension error:
Could not import extension gen_term_table (exception: /<<PKGBUILDDIR>>/sfepy/discrete/common/extmods/cmesh.cpython-38-arm-linux-gnueabi.so: undefined symbol: _aux_hex)
make[3]: *** [Makefile:35: html] Error 2
```

Can trace that back to `mesh.o`, as one would expect:

```
$ readelf -s ./build/temp.linux-i686-3.8/sfepy/discrete/common/extmods/mesh.o | grep aux
   102: 00000000     0 NOTYPE  GLOBAL DEFAULT  UND _aux_hex
```

Marking the private inline functions as `static inline` fixes the issue. Not sure why it only showed up on this function, maybe because it calls another inline function?

Here's a [good guide on inline behaviour in C99](https://www.greenend.org.uk/rjk/tech/inline.html). I think the best solution here is to be far more explicit with inlining. And replace all `inline` keywords with either `static inline` or `extern inline` as appropriate.

--- a/sfepy/discrete/common/extmods/mesh.c
+++ b/sfepy/discrete/common/extmods/mesh.c
@@ -910,13 +910,13 @@ int32 mesh_get_centroids(Mesh *mesh, float64 *ccoors, int32 dim)
   return(RET_OK);
 }
 
-inline float64 _det3x3(float64 j[9])
+static inline float64 _det3x3(float64 j[9])
 {
   return (j[0]*j[4]*j[8] + j[3]*j[7]*j[2] + j[1]*j[5]*j[6]
           - j[2]*j[4]*j[6] - j[5]*j[7]*j[0] - j[1]*j[3]*j[8]);
 }
 
-inline float64 _tri_area(float64 *coors, uint32 *indices, uint32 nc)
+static inline float64 _tri_area(float64 *coors, uint32 *indices, uint32 nc)
 {
 #define VS(ic, id) (coors[nc*indices[ic] + id])
 
@@ -945,8 +945,8 @@ inline float64 _tri_area(float64 *coors, uint32 *indices, uint32 nc)
 #undef VS
 }
 
-inline float64 _aux_hex(float64 *coors, uint32 *indices, uint32 nc,
-                        int32 h, int32 i, int32 j, int32 k)
+static inline float64 _aux_hex(float64 *coors, uint32 *indices, uint32 nc,
+                               int32 h, int32 i, int32 j, int32 k)
 {
 #define VS(ic, id) (coors[nc*indices[ic] + id])
 

